### PR TITLE
Add the ip of the server to the response object

### PR DESCRIPTION
**Pull request by quixoten**
Depends on https://github.com/ruby-protobuf/protobuf/pull/372

### CODE REVIEW

**Comment by quixoten** [comment]
cc @film42

**Comment by film42** [comment]
👍 I like this. Just need to get pb released and fix the gemspec requirement?

File: lib/protobuf/nats/server.rb [hunk 1]
```diff
@@ -22,6 +22,7 @@ def initialize(options)
         @thread_pool = ::Protobuf::Nats::ThreadPool.new(@options[:threads], :max_queue => max_queue_size)
 
         @subscriptions = []
+        @server = options.fetch(:server, first_ipv4_address)
```
Comment (quixoten) [inline lib/protobuf/nats/server.rb]: I'm thinking I should change the default to `Socket.gethostname`. I wanted the ipv4 address to mirror what we have today, but I actually do a reverse lookup on the ipv4 address to in logstash to get the hostname anyways. That lookup has been crashing the QA logstash recently due to timeouts. What say ye?

Comment (quixoten) [inline lib/protobuf/nats/server.rb]: @film42 :point_up:

Comment (film42) [inline lib/protobuf/nats/server.rb]: Sounds good to me.

**Comment by quixoten** [comment]
ready.

**Pull request merged**